Fix "supress" typos

zakariaridouh opened the pull request:
#### a9199700fb693e1d1e27b197280010a483a7500b
<pre>
Fix &quot;supress&quot; typos
<a href="https://bugs.webkit.org/show_bug.cgi?id=267614">https://bugs.webkit.org/show_bug.cgi?id=267614</a>
<a href="https://rdar.apple.com/121478921">rdar://121478921</a>

Reviewed by Tim Nguyen.

Correct misspellings (&quot;supress&quot;) with proper spelling (&quot;suppress&quot;) across codebase.

* LayoutTests/dom/html/level2/html/HTMLTableCellElement21.js:
* LayoutTests/dom/html/level2/html/HTMLTableCellElement22.js:
* LayoutTests/dom/xhtml/level2/html/HTMLTableCellElement21.js:
* LayoutTests/dom/xhtml/level2/html/HTMLTableCellElement22.js:
* LayoutTests/fast/events/popup-blocking-click-in-iframe-expected.txt:
* LayoutTests/fast/events/popup-blocking-click-in-iframe.html:
* LayoutTests/fast/xsl/mozilla-tests.xsl:
* Source/WTF/wtf/PlatformHave.h:
* Source/WebCore/PAL/pal/cocoa/AVFoundationSoftLink.mm:
* Source/WebCore/PAL/pal/spi/cocoa/AVFoundationSPI.h:
* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::isTouchBarUpdateSuppressedForHiddenContentEditable const):
(WebCore::Quirks::isTouchBarUpdateSupressedForHiddenContentEditable const): Deleted.
* Source/WebCore/page/Quirks.h:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
(WebCore::MediaPlayerPrivateAVFoundationObjC::createAVPlayer):
(WebCore::MediaPlayerPrivateAVFoundationObjC::setMuted):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::setIsTouchBarUpdateSuppressedForHiddenContentEditable):
(WebKit::WebPageProxy::setIsTouchBarUpdateSupressedForHiddenContentEditable): Deleted.
* Source/WebKit/UIProcess/WebPageProxy.h:
* Source/WebKit/UIProcess/WebPageProxy.messages.in:
* Source/WebKit/UIProcess/mac/WebViewImpl.mm:
(WebKit::WebViewImpl::updateTouchBar):
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
(WebKit::WebPage::didStartPageTransition):
(WebKit::WebPage::didChangeSelectionOrOverflowScrollPosition):
* Source/WebKit/WebProcess/WebPage/WebPage.h:

Canonical link: <a href="https://commits.webkit.org/280653@main">https://commits.webkit.org/280653@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/74e267154d485ac5d7802dc23fa5d2697d861309

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/57209 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/36537 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/9684 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/60830 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/7652 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/59337 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/44161 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/7842 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/46318 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/5392 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/59239 "Passed tests") | [⏳ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/49396 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/27178 "Passed tests") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/6712 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/6657 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/50301 "Built successfully and passed tests") | [⏳ 🧪 api-ios](https://ews-build.webkit.org/#/builders/API-Tests-iOS-Simulator-EWS "Waiting to run tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/6982 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/62510 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/56451 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/1122 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/7077 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/53579 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/1127 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/49435 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/53652 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12648 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/948 "Passed tests") | | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/78211 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/32366 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/12963 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/33451 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/34536 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/33197 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rniwa commented on Websites/browserbench.org/Speedometer3.0/resources/editors/assets/longscript.js:
```diff
@@ -12256,7 +12256,7 @@
 
   var hydrationParentFiber = null;
```
I don't think we should modify Speedometer 3 content since the benchmark has been released as is. Modifying content even just fixing spelling mistake can have unexpected effects on perf.

rniwa reviewed (changes requested):
Please revert Speedometer 3 changes.

aproskuryakov commented on PerformanceTests/LaunchTime/launch_time.py:
```diff
@@ -12,7 +12,6 @@
 import time
 
 
-
```
This is ok, but not a "suppress" fix.

nt1m reviewed (approved):
Looks good, thanks!

zakariaridouh committed:
Fix "supress" typos
https://bugs.webkit.org/show_bug.cgi?id=267614
rdar://121478921

Reviewed by Tim Nguyen.

Correct misspellings ("supress") with proper spelling ("suppress") across codebase.

* LayoutTests/dom/html/level2/html/HTMLTableCellElement21.js:
* LayoutTests/dom/html/level2/html/HTMLTableCellElement22.js:
* LayoutTests/dom/xhtml/level2/html/HTMLTableCellElement21.js:
* LayoutTests/dom/xhtml/level2/html/HTMLTableCellElement22.js:
* LayoutTests/fast/events/popup-blocking-click-in-iframe-expected.txt:
* LayoutTests/fast/events/popup-blocking-click-in-iframe.html:
* LayoutTests/fast/xsl/mozilla-tests.xsl:
* Source/WTF/wtf/PlatformHave.h:
* Source/WebCore/PAL/pal/cocoa/AVFoundationSoftLink.mm:
* Source/WebCore/PAL/pal/spi/cocoa/AVFoundationSPI.h:
* Source/WebCore/page/Quirks.cpp:
(WebCore::Quirks::isTouchBarUpdateSuppressedForHiddenContentEditable const):
(WebCore::Quirks::isTouchBarUpdateSupressedForHiddenContentEditable const): Deleted.
* Source/WebCore/page/Quirks.h:
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm:
(WebCore::MediaPlayerPrivateAVFoundationObjC::createAVPlayer):
(WebCore::MediaPlayerPrivateAVFoundationObjC::setMuted):
* Source/WebKit/UIProcess/WebPageProxy.cpp:
(WebKit::WebPageProxy::setIsTouchBarUpdateSuppressedForHiddenContentEditable):
(WebKit::WebPageProxy::setIsTouchBarUpdateSupressedForHiddenContentEditable): Deleted.
* Source/WebKit/UIProcess/WebPageProxy.h:
* Source/WebKit/UIProcess/WebPageProxy.messages.in:
* Source/WebKit/UIProcess/mac/WebViewImpl.mm:
(WebKit::WebViewImpl::updateTouchBar):
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
(WebKit::WebPage::didStartPageTransition):
(WebKit::WebPage::didChangeSelectionOrOverflowScrollPosition):
* Source/WebKit/WebProcess/WebPage/WebPage.h:

Canonical link: https://commits.webkit.org/280653@main
